### PR TITLE
Update manifest for a full OS deployment

### PR DESCRIPTION
**Pull request by ghost**
**What is the purpose of this pull request?**:

**What issue does this pull request address?**: Fixes #

**Notes for reviewers to consider**:

**Specific reviewers for pull request**:
@gardlt @drewwalters96

### CODE REVIEW

**Review by drewwalters96 — APPROVED** [review]

File: examples/armada-manifest-v1.yaml [hunk 1]
```diff
@@ -125,41 +147,227 @@ data:
...
 schema: armada/Manifest/v1
 metadata:
   schema: metadata/Document/v1
   name: armada-manifest
 data:
-  release_prefix: armada
-  chart_groups:
-    - openstack-infra-services
-    - openstack-keystone
+  armada:
```
Comment (theyer) [inline examples/armada-manifest-v1.yaml]: @gardlt L368 adds a new level to the Manifest section which will probably break code somewhere. do we want to keep this and adjust code or remove it?

**Review by sh8121att — APPROVED** [review]
LGTM. @intlabs requested this also go in some open PS in review.openstack.org

**Review by intlabs — CHANGES_REQUESTED** [review]
This does not include ceph? Also as @sh8121att pointed out this would be best placed in your open PS in openstack-helm to enable us to perform testing upon it.

**Comment by gardlt** [comment]
This PR is going to openstack-helm. The purpose of this is to display all the functionality of armada in one manifest.

**Comment by theyer** [comment]
@intlabs aside from a one-line change proposed end of day Friday (and the removal of ceph), this was copied directly from the PS which is still up to date.

**Pull request closed without merging**